R/augment.R

```diff
@@ -46,8 +46,10 @@ augment.tune_results <- function(x, parameters = NULL, ...) {
 
   # check/determine best settings
   if (is.null(parameters)) {
-    obj_fun <- .get_tune_metric_names(x)[1]
-    parameters <- select_best(x, metric = obj_fun)
+    metric_info <- choose_metric(x, metric = NULL)
+    metric <- metric_info$metric
+    
```

Comment:
Maybe you are doing things in stages but I think that you have to get/set evaluation time here too: https://github.com/tidymodels/tune/blob/main/R/select_best.R#L82-L85

Can you add non-survival tests here and survival tests in extratests?

Reply:
We'll need an `eval_time` argument for these functions too.

Reply:
`eval_time` argument added here: [`54f0d5b` (#825)](https://github.com/tidymodels/tune/pull/825/commits/54f0d5bfb754fc2aa74217aa54b448c3e744d277)

Reply:
survival tests are done https://github.com/tidymodels/extratests/pull/178

and non-survival tests have been expanded with [`5aefeac` (#825)](https://github.com/tidymodels/tune/pull/825/commits/5aefeacebb6750ae09574a228c71823b651c319f)